themes/default/templates/infopage.html

```diff
@@ -5,7 +5,7 @@
     <div id="menu" class="row" data-bind="visible: infoPage()=='menu', if: infoPage()=='menu'">
         {% include 'menu.html' %}
     </div>
-    <div id="suspend" data-bind="visible: infoPage()=='suspend', if: infoPage()=='suspend'">
+    <div id="suspend" data-bind="visible: infoPage()=='paused' || infoPage()=='resumed', if: infoPage()=='suspend'">
```

Comment:
you'll need to change the `if:` condition to be the same as the one for `visible:`